private-templates-service/adaptivecards-templating-service/src/TemplateServiceClient.ts

```diff
@@ -1091,7 +1166,7 @@ export class TemplateServiceClient {
       });
     });
 
-    router.post("/:id*?", async (req: Request, res: Response, _next: NextFunction) => {
+    router.post("/:id?", async (req: Request, res: Response, _next: NextFunction) => {
```

Comment:
This file is getting bigger. I would consider putting routes to another file. Maybe part of next PRs

Reply:
Ok will do!